### DISCUSSION TITLE
How do I share styles?

### DISCUSSION THREAD

**Discussion by ViktorVorobyev** [lllyasviel/Fooocus#3466] [Q&A] [answered]
![2024-08-07_00-02-32](https://github.com/user-attachments/assets/20309930-4f12-4aa0-bb3a-b72f14f4dd93)
They may be of interest to the community. How do I share it?

**Comment by mashb1t** [comment, marked as answer]
You can create a custom json file, structure see existing styles in sdxl_styles

**Reply by ViktorVorobyev** [reply]
[sdxl_styles_vv.zip](https://github.com/user-attachments/files/16516302/sdxl_styles_vv.zip)
